src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+import { Route, Link } from "react-router-dom";
+import Product from "../Product/Product";
+import "./Introduce.scss";
+
+
+const Introduce = () => {
+  const [input, setInput] = useState("");
+  const [button, setButton] = useState("");
+
+  const HandleInput = (e) => {
+    console.log("input 태그 동작");
+  };
+
+  const Handlebutton = (e) => {
+    console.log("button 태그 동작");
+  };
+
+
+
+  return (
+    <form className="introduce">
+      <div className="profileFrame">
+        <img className="profileImg" />
+        <div className="nameArea">
+          <span className="profileName">신효민</span>
+          <span className="profileSub">44기</span>
+          <span className="inputArea">
+          <input
+            onChange={(e) => {
+              HandleInput(e);
+            }}
+          />
+          <Link to="/product"> <button> 클릭 </button> </Link>
```

Comment:
```suggestion
       <button> 클릭 </button>
```
button에 onClick 이벤트를 이용해서 useNavigate로 product 페이지에 이동할 수 있도록 구현해주세요!